packages/backend/src/server/api/endpoints/admin/accounts/create.ts

```diff
@@ -50,9 +70,21 @@ export default class extends Endpoint<typeof meta, typeof paramDef> { // eslint-
 		private instanceActorService: InstanceActorService,
 	) {
 		super(meta, paramDef, async (ps, _me, token) => {
+			if (ps.initialPassword != null && this.config.initialPassword != null) {
```

Comment:
この書き方だと API 叩くときに `initialPassword` 省略すれば普通に bypass できてしまいそう？
```suggestion
			if (this.config.initialPassword != null) {
```